test_scripts/API/OnSeekMediaClockTimer/ATF_N_OnSeekMediaClockTimer_seekTime_hours_Out_Lower_Bound.lua

```diff
@@ -0,0 +1,94 @@
+--------------------------------------------------------------------------------------------
+-- Requirement summary:
+-- [MOBILE_API] New OnSeekMediaClockTimer notification
+-- [HMI_API] New OnSeekMediaClockTimer notification
+--
+-- Description:
+-- [HMI_API] <param name="hours" type="Integer" minvalue="0" maxvalue="59" mandatory="true">
+-- [MOBILE_API] <param name="hours" type="Integer" minvalue="0" maxvalue="59">
+--
+-- 1. Used preconditions:
+-- a) First SDL life cycle
+-- b) OnSeekMediaClockTimer notification allowed in preloaded file for default app
+-- c) App successfylly registered, consented and activated
+--
+-- 2. Performed steps:
+-- a) HMI sends OnSeekMediaClockTimer notification to SDL with hours put lower bound value ("-1") of seekTime parameter.
```

Comment:
@OVikhrov, please correct step a description with "..**out** lower bound..." Currently it specifies lower bound value